fetch.bs

```diff
@@ -1320,8 +1320,11 @@ cannot be exposed to script. That would violate <a>atomic HTTP redirect handling
 <a for=/>status</a>. Unless stated otherwise it is <code>200</code>.
 
 <p>A <a for=/>response</a> has an associated
-<dfn export for=response id=concept-response-status-message>status message</dfn>.
-Unless stated otherwise it is `<code>OK</code>`.
+<dfn export for=response id=concept-response-status-message>status message</dfn>. Unless stated
+otherwise it is the empty byte sequence.
+
+<p class=note>This does not have a default value of `<code>OK</code>` as otherwise all HTTP/2
+<a for=/>responses</a> would have that value.
```

Comment:
Maybe make it clear that HTTP/2 doesn't support status messages. Something like: "HTTP/2 doesn't support status messages, so this property will be an empty byte sequence for all responses from an HTTP/2 connection"